doc/tutorials/introduction/linux_install/linux_install.markdown

```diff
@@ -21,9 +21,20 @@ The packages can be installed using a terminal and the following commands or by
 Manager:
 @code{.bash}
 [compiler] sudo apt-get install build-essential
-[required] sudo apt-get install cmake git libgtk2.0-dev pkg-config libavcodec-dev libavformat-dev libswscale-dev
+[required] sudo apt-get install cmake git libgtk2.0-dev pkg-config 
 [optional] sudo apt-get install python-dev python-numpy libtbb2 libtbb-dev libjpeg-dev libpng-dev libtiff-dev libjasper-dev libdc1394-22-dev
 @endcode
+Either the libav packages or ffmpeg must be installed, choose either one.
```

Comment:
I think we should give strong recommendations for which platforms it's better to use these libraries. E.g. it's better to use libav on U14 platforms since ffmpeg is not installable from default repositories. The same for U12 and ffmpeg.